### PR TITLE
Add Android 4.4 and Chrome 30 to BrowserStack browsers

### PR DESCRIPTION
**Pull request by joshedney**
## Goal

Add android 4.4, 5.0, 6.0 and Chrome 30, 32, 34, 36, 38, 40, 42 to the list of supported BrowserStack browsers

## Changeset

Add the Nexus 5, Samsung Galaxy S6, S7 device to the list of allowed browsers.
Add Chrome 30, 32, 34, 36, 38, 40, 42 on Windows 7 to the list of allowed browsers

## Tests

Tested locally to ensure device config works.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 6.11.2 - 2022/04/28
```
Comment (twometresteve) [inline CHANGELOG.md:1]: Move to the next minor version for enhancements (i.e. 6.12.0).

Also note that I have a PR in flight at the moment that could go together.

Comment (joshedney) [reply CHANGELOG.md]: Moved version and removed date to wait for your change :)

**Comment by twometresteve** [comment]
The PR comment says there's pipeline coverage to test it, but I don't see a change to the pipeline.  I'd be happy just to test it locally though, or with a temporary change to the pipeline that's then reverted if running on CI is easier.

**Comment by joshedney** [comment]
> The PR comment says there's pipeline coverage to test it, but I don't see a change to the pipeline. I'd be happy just to test it locally though, or with a temporary change to the pipeline that's then reverted if running on CI is easier.

Tested locally to ensure that the device config works

File: lib/maze/browsers_bs.yml [hunk 2]
```diff
@@ -99,6 +120,36 @@ firefox_latest:
...
+
+chrome_34:
+  browser: "chrome"
+  browser_version: "34.0"
+  os: "windows"
+  os_version: "7"
+
+chrome_36:
+  browser: "chrome"
+  browser_version: "35.0"
```
Comment (twometresteve) [inline lib/maze/browsers_bs.yml:143]:
```suggestion
  browser_version: "36.0"
```

**Review by twometresteve — APPROVED** [review]
LGTM

**Pull request merged**